webpack.config.js

```diff
@@ -5,7 +5,7 @@ var webpack = require('webpack');
 
 var base = {
     devServer: {
-        contentBase: path.resolve(__dirname, 'playground'),
+        contentBase: false,
```

Comment:
Correction: I also couldn't find a way to make it provide the build output from `webpack-dev-server` as `/scratch-vm.js` instead of `/playground/scratch-vm.js` _without_ adding the potential to pick up stale files from the `dist` directory.